### PR TITLE
Blobbogram forester

### PR DESCRIPTION
**Pull request by mcguinlu**
- Blobbogram forester

- Fixed documentation

### CODE REVIEW

File: R/rob_blobbogram.R [hunk 1]
```diff
@@ -1,498 +1,290 @@
...
-#'                            font_family = "mono",
-#'                            estimate_col_name = "Estimate"){
-#'
-#'   # This will need to be separated out into its own function, with a good deal of error handling
-#'   table <- merge(data.frame(Study = gsub(",","",rma$slab),
-#'                             yi = rma$yi,
-#'                             vi = rma$vi,
-#'                             est = exp(rma$yi),
-#'                             ci_low = exp(rma$yi - 1.96 * sqrt(rma$vi)),
-#'                             ci_high = exp(rma$yi + 1.96 * sqrt(rma$vi)),
```
Comment (paulsharpeY) [inline R/rob_blobbogram.R:51]: Why exp()?

File: R/rob_blobbogram.R [hunk 2]
```diff
@@ -1,498 +1,290 @@
...
+                                name = "Subtotal",
+                                single_group = FALSE,
+                                add_tests = FALSE,
+                                add_blank = TRUE){
+
+  if (single_group == FALSE) {
+    row <- data.frame(Study = name,
+                      est = exp(rma$b),
+                      ci_low = exp(rma$ci.lb),
+                      ci_high = exp(rma$ci.ub))
```
Comment (paulsharpeY) [inline R/rob_blobbogram.R:71]: Why exp()?

File: R/rob_blobbogram.R [hunk 3]
```diff
@@ -1,498 +1,290 @@
...
+                                    rob_colour = "cochrane",
+                                    rob_tool = "ROB2",
+                                    subset_col_order = NULL){
+
+  table <- merge(data.frame(Study = rma$slab,
+                            yi = rma$yi,
+                            vi = rma$vi,
+                            est = exp(rma$yi),
+                            ci_low = exp(rma$yi - 1.96 * sqrt(rma$vi)),
+                            ci_high = exp(rma$yi + 1.96 * sqrt(rma$vi)),
```
Comment (paulsharpeY) [inline R/rob_blobbogram.R:206]: Why exp()?

**Comment by mcguinlu** [comment]
The functionality in the PR has been replaced by a different and more robust method of achieving the similar figures (`rob_forest`, which uses base graphics via `metafor::forest()` rather than `ggplot2`), and so I am closing it for now.

**Pull request closed without merging**